include/umf.h

```diff
@@ -15,4 +15,13 @@
 #include <umf/mempolicy.h>
 #include <umf/memspace.h>
 
+// Increment the usage reference counter and initialize the global state of libumf
+// if the usage reference counter was equal 0.
+// Returns 0 on success or -1 on failure.
+int umfInit(void);
```

Comment:
Please add an info that this function when exactly user must call this function. Right now it must be called only after dlopen an is not needed in other scenarios, right?

Reply:
Done